src/plugins/Migrations/controllers/commands/migrateGuilds.js

```diff
@@ -0,0 +1,129 @@
+const { Guild, Channel } = require("./../../../../../api/models");
+const migrationsHandler = require("./../../structures/MigrationHandler.js");
+
+exports.help = {
+  name: "migrateGuilds",
+  description: "Migrates all the data for the guilds",
+  usage: "migrateGuilds",
+  example: "migrateGuilds",
+};
+
+exports.conf = {
+  aliases: [],
+  cooldown: 5,
+  permissions: ["ADMINISTRATOR"],
+};
+
+exports.run = async (client, message, args) => {
+  const guilds = client.guilds.cache;
+  for (const guild of guilds) {
+    const guildId = message.guild.id;
+    const guildInDb = await Guild.findOne({ where: { id: guildId } });
+
+    if (guildInDb !== null) {
+      message.channel.send(`${guild[1].name} already has been created`);
+      continue; //Skips to the next guild;
+    }
+
+    /*For things that can be copy-pasted, with no modifications being made to the data,
+        we will use the system below
+      */
+    let keys = [
+      { oldKey: `prefix.${guildId}`, newKey: "prefix" },
+      { oldKey: `${guildId}.newUserRoles`, newKey: "defaultMemberRoles" },
+      { oldKey: `moderation-server.${guildId}`, newKey: "moderationServer" },
+    ];
+
+    const guildObject = migrationsHandler.copyData(keys);
+
+    guildObject.id = guildId;
+
+    const opRoles = [];
+    const modRole = db.get(`moderator.${guildId}`);
+    const adminRole = db.get(`admin.${guildId}`);
+    if (modRole !== null) opRoles.push(modRole);
+    if (adminRole !== null) opRoles.push(adminRole);
+    if (opRoles.length > 0) guildObject.opRoles = opRoles;
+
+    const levels = [];
+    const levelNums = [
+      "one",
+      "five",
+      "ten",
+      "fifteen",
+      "twenty",
+      "twenty-five",
+      "thirty",
+      "thirty-five",
+      "forty",
+    ];
+    for (const levelNum of levelNums) {
+      levels.push({ name: levelNum, role: db.get(`levels.${guildId}.${levelNum}`) });
+    }
+    guildObject.levels = levels;
+
+    const invites = [];
+    const dbInvites = dbh.invite.getInvites(guildId);
+    if (dbInvites) {
+      Object.keys(dbInvites).forEach((inviteName) => {
+        invites.push({ inviteName: inviteName, inviteCode: dbInvites[inviteName] });
+      });
+      if (invites.length > 0) guildObject.invites = invites;
+    }
+
+    const threadInactivityTime = db.get(`thread-inactivity-time.${guildId}`);
+    if (threadInactivityTime) guildObject.threadInactivityTime = threadInactivityTime * 60; //hours to minutes
+
+    const moderationServer = db.get(`moderationServer.${guildId}`);
+    if (moderationServer) guildObject.moderationServer = moderationServer;
+
+    const modCooldown = db.get(`mod-cooldown.${guildId}`);
+    if (modCooldown) guildObject.modCooldown = parseInt(modCooldown);
+
+    const dbGuild = await Guild.create(guildObject);
+
+    await message.channel.send(
+      `Created guild for **${guild[1].name}** with attributes: \`\`\`${Object.keys(
```

Comment:
What does the \`\`\` do? Why do we need the mapping?

Reply:
1) This is markdown https://www.markdownguide.org/extended-syntax/#fenced-code-blocks
It is not necessary, but it makes it more readeable.
2) Because we are looping through the attributes of the guild stored in the database and translating them one by one to text.